src/hooks/chats/venueChat.ts

```diff
@@ -110,12 +89,52 @@ export const useVenueChat = (venueId?: string) => {
     [venueId, userId]
   );
 
+  return {
+    sendMessage,
+    deleteMessage,
+    sendThreadReply,
+  };
+};
+
+const useChatMessages = (venueId?: string) => {
+  const { worldUsersById } = useWorldUsersByIdWorkaround();
+  const { userRoles } = useRoles();
+  const isAdmin = Boolean(userRoles?.includes("admin"));
+  const { userId } = useUser();
+
+  useFirestoreConnect(
+    venueId
+      ? {
+          collection: "venues",
+          doc: venueId,
+          subcollections: [{ collection: "chats" }],
+          orderBy: ["ts_utc", "desc"],
+          storeAs: "venueChatMessages",
+        }
+      : undefined
+  );
```

Comment:
move this to a connect

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/3660c406551764e6c60917460a79a253b51c4a67